<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="EPBYMINW1753" name="Default test" tests="4" failures="1" timestamp="21 Dec 2015 14:16:23 GMT" time="0.094" errors="0">
  <testcase name="@AfterTest logout" time="0.0" classname="by.epam.tat.cherevach.module5.webdriver.GMailOperations">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:182)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:142)
	at org.testng.TestRunner.afterRun(TestRunner.java:1033)
	at org.testng.TestRunner.run(TestRunner.java:636)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1246)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1171)
	at org.testng.TestNG.run(TestNG.java:1066)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:113)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:206)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:177)
]]>
    </failure>
  </testcase> <!-- @AfterTest logout -->
  <testcase name="@AfterTest quit" time="0.0" classname="by.epam.tat.cherevach.module5.webdriver.GMailOperations">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:182)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:142)
	at org.testng.TestRunner.afterRun(TestRunner.java:1033)
	at org.testng.TestRunner.run(TestRunner.java:636)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1246)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1171)
	at org.testng.TestNG.run(TestNG.java:1066)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:113)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:206)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:177)
]]>
    </failure>
  </testcase> <!-- @AfterTest quit -->
  <testcase name="logInGeneral" time="0.0" classname="by.epam.tat.cherevach.module5.webdriver.GMailOperations">
    <skipped/>
  </testcase> <!-- logInGeneral -->
  <testcase name="logInPass" time="0.017" classname="by.epam.tat.cherevach.module5.webdriver.GMailOperations">
    <failure type="org.testng.TestNGException" message="
Can&amp;apos;t invoke public void by.epam.tat.cherevach.module5.webdriver.GMailOperations.logInPass(): either make it static or add a no-args constructor to your class">
      <![CDATA[org.testng.TestNGException: 
Can't invoke public void by.epam.tat.cherevach.module5.webdriver.GMailOperations.logInPass(): either make it static or add a no-args constructor to your class
	at org.testng.internal.Utils.checkInstanceOrStatic(Utils.java:800)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:42)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:643)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:820)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1128)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1246)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1171)
	at org.testng.TestNG.run(TestNG.java:1066)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:113)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:206)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:177)
]]>
    </failure>
  </testcase> <!-- logInPass -->
  <testcase name="sendEmail" time="0.0" classname="by.epam.tat.cherevach.module5.webdriver.GMailOperations">
    <skipped/>
  </testcase> <!-- sendEmail -->
  <testcase name="emailReceived" time="0.0" classname="by.epam.tat.cherevach.module5.webdriver.GMailOperations">
    <skipped/>
  </testcase> <!-- emailReceived -->
</testsuite> <!-- Default test -->
